src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall().toString());
```

Comment:
checklist: Don’t create several instances of the same class if you can use only one instance for your purposes[](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes)
```suggestion
            System.out.println(new Lottery().getRandomBall());
```

we don't need to explicitly call the toString() method here, this happens under the hood